imaging/loaders/commonLoader.ts

```diff
@@ -273,3 +269,80 @@ export const getSopInstanceUIDFromLarvitarManager = function (
     key => series.instanceUIDs[key] === imageId
   );
 };
+const getTemporalSeriesData = function( series: Series,) {
+  type RecursivePartial<T> = {
+    [P in keyof T]?: RecursivePartial<T[P]>;
+  };
+  type SeriesData = StoreViewport;
+  const data: RecursivePartial<SeriesData> = {};
+  if (series.is4D) {
+    data.isMultiframe = false;
+    data.isTimeserie = true;
+    // check with real indices
+    data.numberOfSlices = series.numberOfImages;
+    data.numberOfTemporalPositions = series.numberOfTemporalPositions;
+    data.imageIndex = 0;
+    data.timeIndex = 0;
+    data.imageId = series.imageIds[data.imageIndex];
+    data.timestamp = series.instances[data.imageId].metadata[
+      "x00080033"
+    ] as number;
+    data.timestamps = [];
+    data.timeIds = [];
+    each(series.imageIds, function (imageId: string) {
+      (data.timestamps as any[]).push(
+        series.instances[imageId].metadata.contentTime
+      );
+      (data.timeIds as any[]).push(
+        series.instances[imageId].metadata.temporalPositionIdentifier! - 1 // timeId from 0 to N
+      );
+    });
+  }
+  return data as SeriesData;
+};
+/**
+ * Return the SOP Instance UID of a specific imageId stored in the Larvitar Manager
+ * @instance
+ * @function updateViewportDataInLarvitarManager
+ * @param {Series} seriesStack The Id of the series
+ * @param {String} elementId The Id of the image
+ */
+export const updateViewportDataInLarvitarManager = function(
```

Comment:
anche questa la metterei in imageRendering, in quanto non c'entra nulla con questo file (dicom common loader)